### PR TITLE
GUACAMOLE-377: Add client-side support for frame tracking and RemoteFX parameter.

### PR DESCRIPTION
**Pull request by mike-jumper**
Corresponding to apache/guacamole-server#363, this change adds support for enhanced frame tracking, an extension which implements that tracking and renders performance metrics in a toolbar, and the usual parameter definitions and translations.

### CODE REVIEW

File: extensions/guacamole-display-statistics/src/main/resources/directives/guacClientStatistics.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * A directive which displays one or more Guacamole clients in an evenly-tiled
```
[the remark this answers is not in the document]
Comment (mike-jumper) [reply extensions/guacamole-display-statistics/src/main/resources/directives/guacClientStatistics.js:21]: Oops - that's some unfinished copy pasta for sure. I'll fix.

**Comment by jmuehlner** [comment]
What does the toolbar actually look like? Can you post a screenshot to this PR?

**Comment by mike-jumper** [comment]
Sure - I'll deploy a copy and grab a screenshot.

**Review by jmuehlner — APPROVED** [review]

**Comment by jmuehlner** [comment]
Awesome! Looks great.

**Pull request merged**